geoana/em/fdem/layered.py

```diff
@@ -121,7 +121,7 @@ def magnetic_field(self, xyz, field="secondary"):
         rTE = rTE.reshape((n_frequency, *lambd.shape))
 
         # secondary is height of receiver plus height of source
-        rTE *= np.exp(-lambd*(xyz[:, -1] + h)[:, None])
+        rTE *= np.exp(-lambd*(dxyz[:, -1] + 2*h)[:, None])
```

Comment:
These two lines are equivalent to each other (dxyz + 2*h) = ((xyz - location) + 2*location)=(xyz + location)

In my opinion, the original is clearer.

Reply:
Sure. I'll change it back